tools/wasm-sourcemap.py

```diff
@@ -352,8 +371,8 @@ def main():
 
   logger.debug('Saving to %s' % options.output)
   map = build_sourcemap(entries, code_section_offset, options)
-  with open(options.output, 'w') as outfile:
-    json.dump(map, outfile, separators=(',', ':'))
+  with open(options.output, 'w', encoding='utf-8') as outfile:
+    json.dump(map, outfile, separators=(',', ':'), ensure_ascii=False)
```

Comment:
So `ensure_ascii=False` means that non-ascii chars are not escaped as they normally would be.    Is that what we want for source maps?